### DISCUSSION TITLE
Production Environment Setup

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [neonexus/sails-react-bootstrap-webpack#89] [General]
Hi @neonexus,

Greetings. I would like your opinion on my plan for production environment setup for the Sails-React web app.

Maximum number of users that could access the webapp individually or in parallel would be not more than 150 users.

Ideally, I would use docker for production. But now, I am deploying the webapp in a Windows VM due to some reasons. So, I planned to use NGINX for serving React frontend while [pm2](https://pm2.keymetrics.io/) would handle the backend. 

What do you think of this? Do you have any better suggestions for deploying in a Windows VM?

Thank you!

**Comment by neonexus** [comment]
I don't work with Windows, for a lot of reasons, so I'm afraid I'm not the best source to ask about that. That said, your plan sounds very similar to what I would do with Docker.
